script.js

```diff
@@ -1,11 +1,27 @@
 // Create an array called `colors` that contains three colors.
 
+var colors = ['pink', 'purple', 'blue'];
+
 // Log the last item in the array to the console.
 
+console.log(colors[2])
+
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
 
+for (var i = 0; i < colors.length; i++) {
+  console.log("My favorite color is " + colors[i])
+}
+
 // Create an `instructor` object that contains three key-value pairs.
 
+var instructor = {
+hello : "world",
+First_name: "Juliet",
+Last_name: "Berman"
+}
```

Comment:
try and follow style guides for variable/property syntax. ie camelCased and starts with lower case letter:

```js
firstName: "Juliet"
```